.github/workflows/quay-mgmt.yml

```diff
@@ -18,7 +23,10 @@ jobs:
         uses: ansible/ansible-lint-action@master
```

Comment:
It looks like the author(s) of this action have changed to using `main` as their default branch.
```
Error: Unable to resolve action `ansible-community/ansible-lint-action@master`, unable to find version `master`
```
So we can pin it to a release or it needs to be changed to `ansible/ansible-lint-action@main`

Reply:
corrected